src/App.css

```diff
@@ -12,7 +12,7 @@
 
 @layer base {
   .heading1 {
-    @apply text-56 leading-72 text-white paytone;
+    @apply text-56 leading-72 text-white paytone text-center;
```

Comment:
this may add text-center as a style to every heading1 throughout the app, rather than just the hero section? can we delegate this to a different part of the code that's more specific to the hero?